exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -792,18 +792,28 @@ func (c *CoinbasePro) GetOrderHistory(ctx context.Context, req *order.GetOrdersR
 		return nil, err
 	}
 	var respOrders []GeneralizedOrderResponse
-	for i := range req.Pairs {
-		fpair, err := c.FormatExchangeCurrency(req.Pairs[i], asset.Spot)
-		if err != nil {
-			return nil, err
+	if len(req.Pairs) > 0 {
```

Comment:
Could you also please update `TestGetOrderHistory` and have a scenario where `Pairs` len is 0?

Reply:
Hi, the else case for if len(req.Pairs) > 0 should serve as scenario for len is 0 and nil since nil also mean len not > 0, which will go into the else branch and sending request to Coinbase Pro without specifying the product_id query param and get order history across all markets.

Reply:
Sorry khcchiu, I meant to say, could you add a test scenario in `TestGetOrderHistory` under `coinbasepro_test.go` where you call `GetOrderHistory` while passing in nil `Pairs` to add test coverage to the scenario you just added.
At the moment, the test `TestGetOrderHistory` only calls:
```
	var getOrdersRequest = order.GetOrdersRequest{
		Type:      order.AnyType,
		AssetType: asset.Spot,
		Pairs:     []currency.Pair{testPair},
	}

	_, err := c.GetOrderHistory(context.Background(), &getOrdersRequest)
```
and I would like to see another call with:
```
	getOrdersRequest.Pairs = nil
	_, err = c.GetOrderHistory(context.Background(), &getOrdersRequest)
```

Reply:
Sure, added scenarios with empty and nil pairs in test